My New File

```diff
@@ -6,3 +6,8 @@ It is about checking to see if we can have a private repository
 Let's see what we can do
 
 Adding some more content here
+
```

Comment:
Need to add another line here to say what we are going to do

Reply:
Can you pls let me know what to add

Reply:
Add the following line:

We are going to start with bla bla bla